<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFRDRCurveDataEditor</class>
 <widget class="QWidget" name="QFRDRCurveDataEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QFRDRCurveDataEditor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Curve:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbCurve"/>
     </item>
     <item>
      <widget class="QCheckBox" name="chkOverlay">
       <property name="text">
        <string>overlay all curves from this RDR</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFPlotter" name="plotter" native="true"/>
   </item>
   <item>
    <widget class="QGroupBox" name="grpProperties">
     <property name="title">
      <string>Curve Properties</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="edtName"/>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>   type:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbType">
          <item>
           <property name="text">
            <string>points</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>lines</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>histogram/bars</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>axes:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="chkLogX">
          <property name="text">
           <string>log. x-axis</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkLogY">
          <property name="text">
           <string>log. y-axis</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>axis labels: x:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtLabelX"/>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>   y:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtLabelY"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QFPlotter</class>
   <extends>QWidget</extends>
   <header>qfplotter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
